<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Januar 1 2020/2021</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="popuniButton">
         <property name="text">
          <string>Popuni izvore</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="izvoriLabel">
         <property name="text">
          <string>Izvori</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="izvoriList"/>
       </item>
       <item>
        <widget class="QLabel" name="ukupnaZapreminaIzvoraLabel">
         <property name="text">
          <string>Ukupna zapremina izvora</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ukupnaZepreminaIzvoraLineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="transferButton">
         <property name="text">
          <string>Zapocni transfer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ukupnaZapreminaRezervoaraLabel">
         <property name="text">
          <string>Ukupna zapremina razervoara</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ukupnaZapreminaRezervoaraLineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="tekucaZepreminaRezervoaraLabel">
         <property name="text">
          <string>Tekuca zepremina rezervoara</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tekucaZapreminaLineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="gubitakLabel">
         <property name="text">
          <string>Gubitak</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="gubitakLineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="procitajXML">
       <property name="text">
        <string>Procitaj XML</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="procitajJSON">
       <property name="text">
        <string>Procitaj JSON</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="procitaj01">
       <property name="text">
        <string>Procitaj 01</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="ispisiXML">
       <property name="text">
        <string>Ispisi XML</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ispisiJSON">
       <property name="text">
        <string>Ispisi JSON</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ispisi01">
       <property name="text">
        <string>Ispisi 01</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
